cdn/feeder-cdn.yml

```diff
@@ -0,0 +1,69 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Description: >
+  Feeder bucket CDN (or really any plain S3-bucket CDN)
+Metadata:
+  AWS::CloudFormation::Interface:
+    ParameterGroups:
+      - Label:
+          default: Distribution Parameters
+        Parameters:
+          - OriginBucket
+          - DistributionDomain
+          - CertificateArn
+    ParameterLabels:
+      OriginBucket:
+        default: Origin S3 Bucket
+      DistributionDomain:
+        default: Domain Name
+      CertificateArn:
+        default: ACM Certificate Arn
+Parameters:
+  OriginBucket:
+    Type: String
+    Description: eg. staging-prx-feed.s3.amazonaws.com, prx-feed.s3.amazonaws.com
+  DistributionDomain:
+    Type: String
+    Description: eg. f-staging.prxu.org, f.prxu.org
+  CertificateArn:
+    Type: String
+    Description: eg. arn:aws:acm:<region>:<account>:certificate/<guid>
+Resources:
+  CloudFrontDistribution:
+    Type: "AWS::CloudFront::Distribution"
+    Properties:
+      DistributionConfig:
+        Aliases:
+          - !Ref DistributionDomain
+        CustomErrorResponses:
+          - ErrorCachingMinTTL: 0
+            ErrorCode: 403
+            ResponseCode: 404
+            ResponsePagePath: /404.html
+        DefaultCacheBehavior:
+          AllowedMethods: [HEAD, GET]
+          CachedMethods: [HEAD, GET]
+          Compress: false
+          DefaultTTL: 86400
+          ForwardedValues:
+            QueryString: false
+          TargetOriginId: feeder-s3-bucket
+          ViewerProtocolPolicy : allow-all
+        DefaultRootObject: index.html
+        Enabled: true
+        HttpVersion: http2
+        IPV6Enabled: true
+        Origins:
+          - DomainName: !Ref OriginBucket
+            Id: feeder-s3-bucket
+            S3OriginConfig: {}
+        PriceClass: PriceClass_200
+        ViewerCertificate:
+          AcmCertificateArn: !Ref CertificateArn
+          SslSupportMethod: sni-only
+      Tags:
+        - Key: Project
+          Value: Dovetail
```

Comment:
This should probably be `feeder.prx.org`

Reply:
Hmm... I think I pulled that tag from the source S3 bucket.  (`prx-feed.tag.Project == "Dovetail").  But I'm fine with feeder.  Should that be capitalized "Feeder" though?

Reply:
Oh hm. Maybe that should change too, if the bucket only has feeds from feeder. I'm not sure about `Feeder` vs `feeder.prx.org`. We're somewhat inconsistent between projects, and my phone is in the other room so I couldn't log in to check.

Reply:
The ALBs have `Project = feeder` (lowercase) ... so I'll go with that!